mrtt-ui/src/data/questions.js

```diff
@@ -435,7 +435,8 @@ const costs = {
       'Fiscal ',
       'Risk management ',
       'Regulatory ',
-      'Unknown'
+      'Unknown',
+      'Other'
```

Comment:
This other doesnt need clarification like the other others?

Reply:
I wasn't 100% sure on this one - It is probably worth adding the text input for clarification